base/pkg/entry.jl

```diff
@@ -11,6 +13,7 @@ macro recover(ex)
         try $(esc(ex))
         catch err
             show(err)
+            print('\n')
```

Comment:
`show` just prints string as it is. It doesn't process whitespace chars. Try `show("\n")`